BookingsApi/BookingsApi.Domain/JudiciaryParticipant.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using BookingsApi.Domain.Enumerations;
+
+namespace BookingsApi.Domain
+{
+    public class JudiciaryParticipant : ParticipantBase
+    {
+        private readonly DateTime _currentUTC = DateTime.UtcNow;
```

Comment:
I didn't realise we don't have a tracking (created/updated) column for all the other participant types. Not sure if there was a specific reason for adding it to the JudicaryParticipant; but if not, then perhaps we should have the participant base inherit the TackableEntity class. Although just a suggestion, feel free to ignore if you think it adds too much scope creep.

Reply:
that was my original plan (I had JudiciaryParticipant inheriting from TrackableEntity) however the Participant class has the datetime fields as non-nullable, whereas the TrackableEntity has them as nullable, so changing ParticipantBase to inherit from the TrackableEntity would result in a migration to change the Participant datetime columns to nullable.

it's a good spot though. I wonder if it would be better to have a separate tech debt ticket to standardise these trackable properties so that they are consistent across the entities that need them.

Reply:
Ok sounds good, ill add a tech-debt ticket for this

Reply:
ah thanks for that Will